test/ng/directive/booleanAttrsSpec.js

```diff
@@ -195,23 +207,23 @@ describe('ngSrc', function() {
           // then calling element.setAttribute('src', 'foo') doesn't do anything, so we need
           // to set the property as well to achieve the desired effect
 
-          var element = $compile('<div ng-src="{{id}}"></div>')($rootScope);
+          var element = $compile('<img ng-src="{{id}}"></img>')($rootScope);
 
           $rootScope.$digest();
-          expect(element.prop('src')).toBeUndefined();
+          expect(element.prop('src')).toBe('');
           dealoc(element);
 
-          element = $compile('<div ng-src="some/"></div>')($rootScope);
+          element = $compile('<img ng-src="some/"></img>')($rootScope);
 
           $rootScope.$digest();
-          expect(element.prop('src')).toEqual('some/');
+          expect(element.prop('src')).toContain('some/');
           dealoc(element);
 
-          element = $compile('<div ng-src="{{id}}"></div>')($rootScope);
+          element = $compile('<img ng-src="{{id}}"></img>')($rootScope);
           $rootScope.$apply(function() {
             $rootScope.id = $sce.trustAsResourceUrl('http://somewhere');
           });
-          expect(element.prop('src')).toEqual('http://somewhere');
+          expect(element.prop('src')).toEqual('http://somewhere/');
```

Comment:
It is adding the `/` because that is how the empty "path" is determined by the url resolver.
I changed it to 

```
$rootScope.$apply(function() {
            $rootScope.id = $sce.trustAsResourceUrl('http://somewhere/abc');
          });
          expect(element.prop('src')).toEqual('http://somewhere/abc');
```

which shows that we don't arbitrarily add a slash to the end of the path